src/index.test.js

```diff
@@ -240,7 +231,8 @@ test('reads text from fs if filePath is provided but not text', () => {
   } catch (e) {
     // ignore
   }
-  expect(fsMock.readFileSync).toHaveBeenCalledTimes(1)
+  // Two hits to .eslintignore
```

Comment:
huh... Weird. I'm not sure I understand what's going on here, but I think I'm ok with it as is. Thanks!